packages/compass-e2e-tests/tests/instance-databases-tab.test.ts

```diff
@@ -151,6 +151,9 @@ describe('Instance databases tab', function () {
 
     // Drop it and refresh again
     await browser.shellEval(`db.dropDatabase();`);
+    // looks like if you refresh too fast the database appears in the list but
+    // the stats never load, so just pause for bit first
+    await browser.pause(1000);
```

Comment:
It kills me having to add a pause, but can't think of anything else.

Reply:
Aaaand it is still flaking: https://parsley.mongodb.com/evergreen/10gen_compass_main_rhel_test_packaged_app_40x_community_patch_5e921b4a43f0a6be6db48796fd761fd2139b80f0_65c6039a57e85ac52afdb5ef_24_02_09_10_51_08/0/task?bookmarks=0,53930&shareLine=53480

https://mciuploads.s3.amazonaws.com/10gen-compass-main/5e921b4a43f0a6be6db48796fd761fd2139b80f0_leroux.bodenstein_251/rhel/test-packaged-app-40x-communityscreenshot-compass-e2e-tests__Instance-databases-tab-can-refresh-the-list-of-databases-using-refresh-controls.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAJNLNT5VZ22YB4GWQ%2F20240209%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20240209T115304Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-Amz-Signature=6ad340ec59f76b808ff9b46ca6b400b64317d3396dc4bcb3959845daff788d2c

Reply:
Pushed some more commits to at least debug this next time this flakes. I've only seen it on RHEL76 and only in CI.